client/main/main.js

```diff
@@ -160,7 +160,32 @@ $(document).ready(() => {
     backstageModel.on('change:activated', function(event) {
         mainTable.setActive( ! event.changed.activated);
         if (event.changed.activated === false)
-            ribbonModel.set('selectedIndex', 1);
+            ribbonModel.set('selectedTab', 'analyse');
+    });
+
+    ribbonModel.on('actionRequest', (action) => {
+        if (action.tabName === 'data') {
+            if (action.name === 'editVar') {
+                if (dataSetModel.get('editingVar') === null)
+                    dataSetModel.set('editingVar', mainTable.selection.colNo);
+                else
+                    dataSetModel.set('editingVar', null);
+            }
+            else if (action.name === 'paste') {
+                let promise = mainTable._pasteClipboardToSelection();
```

Comment:
if you're calling these from outside the class you should probably make them public (remove the underscore).